src/lib/tokens.js

```diff
@@ -1,19 +1,46 @@
 // Create an error to throw if needed
-const error = new Error('Missing Auth Token!');
+const error = new Error('Bad Auth Token!');
```

Comment:
Another translated string

Reply:
Also, I can't say I've ever seen anyone reuse an Error Object. Should there be any gotchas with that?